src/app/layouts/header/export-popup/export-popup.component.ts

```diff
@@ -14,9 +14,18 @@ export class ExportPopupComponent implements OnInit, OnDestroy {
     @ViewChild('popup') popup: PopupComponent;
     schemeName: string;
     makeSwatch = false;
+    saveAsFile = true;
+    showFileContent = false;
+    fileContent: string;
     outputFile: string;
     subscription: Subscription;
     showOutputFile: boolean;
+    loadIndicatorVisible = false;
+    buttonDisabled: boolean;
```

Comment:
We have several buttons in this component. Let's name this variable like applyButtonDisabled.